packages/api/contract.go

```diff
@@ -52,9 +52,16 @@ func contract(w http.ResponseWriter, r *http.Request, data *apiData, logger *log
 	}
 	info := (*contract).Block.Info.(*script.ContractInfo)
 
+	var signedBy int64
```

Comment:
seems like this variable is useless, signID is enough

Reply:
signedBy can be params[`signed_by`] or zero.
signID  can be params[`signed_by`] or data.keyID.
They are the same only when params[`signed_by`] is defined.